### PR TITLE
fix(multiselect): clear button position

### PR DESCRIPTION
**Pull request by inikolova**
Closes telerik/kendo-theme-default#575

### CODE REVIEW

**Review by joneff — APPROVED** [review]

**Comment by joneff** [comment]
@inikolova I've been thinking on this... the best way to position the clear button, in this particular case is to align it centrally in a box that's as big as the tokens and place the box the same distance from the ledge:

image

**Comment by inikolova** [comment]
cc @yordanov

**Comment by yordanov** [comment]
Yep, as soon as these dimensions are kept:

dimensions

**Comment by joneff** [comment]
@yordanov we can't position the edge of an icon. Instead, we can position the icon container.

**Comment by yordanov** [comment]
Icon's placeholder width x height is 16px  x 16px. As seen on the image above: 
Position of icon's **center** from **top**:  6px + 16/2px = **14px**
Position of icon's **center** from **right**: 6px + 16/2px = **14px**

Position of icon's placeholder top-right edge: top: 6px; right: 6px;

File: scss/multiselect/_layout.scss [hunk 1]
```diff
@@ -9,6 +9,11 @@
             visibility: hidden;
         }
 
+        .k-clear-value {
+            top: $padding-x-lg / 2;
```
Comment (gyoshev) [inline scss/multiselect/_layout.scss:13]: The value fixes the issue, but consider using a `y`-unit when calculating vertical offset. If left like this, changing the `padding-x-lg` will offset the icon vertically, which is not intuitive for the developers who use the theme.

**Comment by gyoshev** [comment]
I agree with @joneff about changing the icon layout, as it may result in less styles overall and also allows to increase the button hit area (which is currently 16px x 16px, requires some precision). Since it is likely to require a rendering change it may take some time to propagate across the components, we can fix the issue with this PR and improve upon it in another.

@yordanov do you think that the hit area is adequate for the button?

**Comment by yordanov** [comment]
Yep, the bigger the hit area - the better.  Token sized clear button looks fine to me.

**Pull request merged**